### PR TITLE
Update property-services-registry.yml

### PR DESCRIPTION
**Pull request by shubhang-eGov**

### CODE REVIEW

**Review by shashwat-egov — APPROVED** [review]

File: egov-persister/property-services-registry.yml [hunk 1]
```diff
@@ -19,7 +25,7 @@ serviceMaps:
         dbType: JSONB
 
     - query: INSERT INTO eg_pt_property (id, propertyid, tenantid, surveyid, accountid, oldpropertyid, status, acknowldgementnumber, propertytype, ownershipcategory, usagecategory, creationreason, nooffloors, landarea, superbuiltuparea, linkedproperties, source,  channel, createdby, createdtime, lastmodifiedby, lastmodifiedtime, additionaldetails) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);
-      basePath: Property
+      basePath: $.Property
```
Comment (talele08) [inline egov-persister/property-services-registry.yml:28]: Why is this change needed?

**Review by talele08 — CHANGES_REQUESTED** [review]
Deploy the updated build of persister which has changes related to signed audit and then only merge the PR else it will break. Take the branch from @shashwat-egov

File: egov-persister/property-services-registry.yml (hunk 1 above)
Comment (shubhang-eGov) [reply egov-persister/property-services-registry.yml:28]: shashwat asked me to make this change by referring to this branch: https://github.com/egovernments/configs/blob/core-113/egov-persister/property-services-registry.yml
This branch is not yet merged, so  I think he asked me to replicate these changes so the code won't break.

**Review by talele08 — APPROVED** [review]
